locales/de/common.json

```diff
@@ -274,6 +269,131 @@
+  "Check for mod updates": "Nach Updates für Mods suchen",
+  "Provide Feedback": "Feedback senden",
+  "Describe in detail what you were doing and the feedback you would like to submit.": "Beschreibe detailliert, was Du getan hast und was Du gerne berichten möchtest.",
+  "Your Message": "Deine Nachricht",
+  "E.g.:\nSummary: The mod downloads properly but when I try to install it nothing happens.\nExpected Results: The mod is installed.\nActual Results: Nothing happens.\nSteps to reproduce: Download a mod, then click Install inside the Actions menu.": "Z.B.:\nZusammenfassung: Die Mod wird korrekt heruntergeladen, aber nichts passiert, wenn ich sie installieren will.\nErwartetes Verhalten: Die Mod wird installiert.\nTatsächliches Verhalten: Nichts passiert.\nZum Reproduzieren: Eine Mod herunterladen, dann im \"Aktionen\"-Menü auf Installieren klicken.",
+  "Select file to attach": "Datei als Anhang auwählen",
+  "or": "oder",
+  "Attach Special File": "Spezielle Datei anhängen",
+  "System Information": "Systeminformationen",
+  "Vortex Log": "Vortex-Log",
+  "Application Settings": "Anwendungseinstellungen",
+  "Application State": "Status der Anwendung",
+  "Send anonymously": "Anonym versenden",
+  "Submit Feedback": "Feedback versenden",
+  "feedback-instructions": "Bitte<1></1><2><0>verwende Satzzeichen und Zeilenumbrüche,</0><1>sei präzise und themabezogen. Du musst keine ganzen Sätze bilden. Ein Bugreport ist ein technisches Dokument, keine Prosa,</1><2>melde nur einen Fehler pro Nachricht,</2><3>stelle keine eigenen Vermutungen auf, melde einfach, was Du siehst und was Du erwartet hast,</3><4>benutze, wenn möglich, Beispiele, wie man den Fehler nachbilden kann. Auch bei einem generellen Problem(\"Mod-Installer verwenden dieses Feature, wenn sie jenes verwenden sollten\"), beschreibe eine Möglichkeit, das Problem sichtbar zu machen.</4></2>Die meiste Zeit wird üblicherweise gebraucht, wenn wir versuchen, einen Fehler zu reproduzieren, um ihn fixen zu können. Je weniger Zeit wir darauf verschwenden, desto mehr Zeit haben wir, um tolle neue Features zu programmieren!",
```

Comment:
I personally would add something like "Bitte schreibe dein Feedback in Englisch", because although Tannin, you and me can speak it, other people who might read the Messages can not.

Reply:
Good point, I may not be the only person reading that feedback

Reply:
In that case, how do I use those xml(?) tags in order to add another line to the list in `feedback-instructions`?

Reply:
Hmm, I'm not sure you can tbh.
In the original text those are html tags (br, ul, li) but the i18next library seems to translate them.
I wasn't able to find any documentation how that works but it appears the tags are mapped to numbers in the order they appear in the original which means adding a tag may not be possible at all, because it wouldn't know what html tag it's supposed to be.
I'll see if we can influence that mapping somehow

Reply:
I mean, we could just put it there in the English version as well. Or just translate it to an empty string in the English *commons.json*.

Reply:
Yes, we could, though tbh. I've not received a single non-english feedback anyway so this at least seems to be self-explanatory to most. I'd still like to understand what's going on with the translation lib and html as this may get in our way in the future.